File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/raul-progr.py
#codigo fizzbuzz

for i in range(1, 100):
    if (i % 3 == 0 and i % 5 == 0):
        print("fizzbuzz")
    else:
        if (i % 3 == 0):
            print("fizz")
        else:
            if (i % 5 == 0):
                print("buzz")
            else:
                print(i)

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/javascript/raul-progr.js
for (i = 0; i <= 100; i++){
    if (i % 3 === 0 && i % 5 === 0){
        console.log("Fizzbuzz")
    } else if (i % 5 === 0){
        console.log("Buzz")
    } else if (i % 3 === 0){
        console.log("Fizz")
    } else {
        console.log(i)
    }
}